### DISCUSSION TITLE
用不合法密码也能注册，但是无法登录了

### DISCUSSION THREAD

**Discussion by wwawwaww** [Liu233w/acm-statistics#2660] [Bug reports]
密码只用数字也能注册，退出账号后无法登录

**Comment by Liu233w** [comment]
感谢。表单验证的问题我已经在 #2661 里修复了。但是经过我的测试，不合法的密码在注册成功之后是可以登录的。您能在详细测一下吗？

**Reply by wwawwaww** [reply]
刚才用新号试了一下确实可以了，但是之前的号浏览器自动保存的密码登不上，也许是我手滑了

**Reply by wwawwaww** [reply]
找到大概原因了，注册时浏览器保存的是验证的密码(第二行)，登录要是第一次输的密码，即使两次密码不一样也能注册成功

**Reply by Liu233w** [reply]
> 找到大概原因了，注册时浏览器保存的是验证的密码(第二行)，登录要是第一次输的密码，即使两次密码不一样也能注册成功

确实有这个问题。等那个PR合并之后应该就能修复了。

**Comment by Liu233w** [comment]
@all-contributors please add @wwawwaww  for bugs
